crates/shared/src/http_solver/model.rs

```diff
@@ -219,7 +221,12 @@ pub struct SettledBatchAuctionModel {
     pub submitter: SubmissionPreference,
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(flatten)]
+    /// TODO remove this field once all solvers conform to sending the
+    /// success_probability
     pub score: Option<Score>,
+    /// The probability of the settlement to be mined successfully.
+    /// Expected values [0..1]
+    pub success_probability: Option<f64>,
```

Comment:
Didn't we also want to add an optional gas estimate here?

Reply:
Added `gas_cost` which is `gas_estimate*gas_price`.